test/it/uk/gov/hmrc/exports/migrations/changelogs/cache/RenameAndConvertBorderTransportNationalityFieldISpec.scala

```diff
@@ -0,0 +1,1185 @@
+package uk.gov.hmrc.exports.migrations.changelogs.cache
+
+import uk.gov.hmrc.exports.base.IntegrationTestMigrationToolSpec
+import uk.gov.hmrc.exports.services.CountriesService
+
+import javax.inject.Inject
+
+class RenameAndConvertBorderTransportNationalityFieldISpec @Inject() (countriesService: CountriesService) extends IntegrationTestMigrationToolSpec {
+
+  override val collectionUnderTest = "declarations"
+  override val changeLog = new RenameAndConvertBorderTransportNationalityField(countriesService)
+
+  "RenameAndConvertBorderTransportNationalityField migration" should {
+
+    import RenameAndConvertBorderTransportNationalityFieldISpec._
+
+    "correctly migrate declarations renaming countryName field and converting contents to country codes" in {
+      runTest(declarationBeforeMigration1, declarationAfterMigration)
+    }
+
+    "rename countryName field but leave contents when already a country code " in {
+      runTest(declarationBeforeMigration2, declarationAfterMigration)
+    }
+
+    "not alter documents already migrated" in {
+      runTest(declarationAfterMigration, declarationAfterMigration)
+    }
+  }
```

Comment:
Need to test the 'not migrated but containing a none-mapable country value' scenario

Reply:
Added